navbar.js

```diff
@@ -0,0 +1,49 @@
+// OPEN/CLOSE HAMBURGER MENU
+
+// const hamburger = document.querySelector('.hamburger');
+// const close = document.querySelector('.closenav');
+// const navList = document.querySelector('.navList');
+// const navLink = document.querySelectorAll('.nav-link');
+// hamburger.addEventListener('click', () => {
+//   nav.classList.add('active');
+// });
+
+// close.addEventListener('click', () => {
+//   nav.classList.remove('active');
+// });
+
+// navLink.forEach((element) => element.addEventListener('click', () => {
+//   nav.classList.remove('active');
+// }));
```

Comment:
- [ ] [REQRUIED] Kindly, try not to use commented code in your code base. It makes the whole code base unclean and hard to maintain. Instead you can use `git reset` for that manner.